featuremanagement/aio/_defaultfilters.py

```diff
@@ -43,11 +26,12 @@ async def evaluate(self, context, **kwargs):
         :return: True if the current time is within the time window
         :rtype: bool
         """
-        return super().evaluate(context, **kwargs)
+        time_window_filter = SyncTimeWindowFilter()
+        return time_window_filter.evaluate(context, **kwargs)
```

Comment:
nit: can this be in-line?

Reply:
We should actually probably not create a new instance every time.